cfme/tests/containers/test_relationships_tables.py

```diff
@@ -8,252 +7,73 @@
-
-
-@pytest.mark.meta(blockers=[1365878])
-@pytest.mark.parametrize('rel, detailfield', [
-    ('Containers Provider', 'Name'),
-    ('Image Registry', 'Host'),
-    ('Projects', None),
-    ('Pods', None),
-    ('Containers', None),
-    ('Nodes', None)
-])
-def test_images_rel(provider, rel, detailfield):
-    """ https://bugzilla.redhat.com/show_bug.cgi?id=1365878
     """
-    # Nav to provider first
-    navigate_to(provider, 'Details')
-
-    # Then to container images for that provider
-    # Locate Relationships table in provider details
-    images_key = ({
-        version.LOWEST: 'Images',
-        '5.7': 'Container Images'
-    })
-    sel.click(InfoBlock.element('Relationships', version.pick(images_key)))
-
-    # Get the names of all the container images from the table
-    list_tbl_image = CheckboxTable(
-        table_locator="//div[@id='list_grid']//table")
-    ui_images = [r.name.text for r in list_tbl_image.rows()]
-
-    for name in ui_images:
-        img = Image(name, provider)
-        navigate_to(img, 'Details')
-
-        val = img.get_detail('Relationships', rel)
-        assert val != 'Unknown image source'
-
-        sel.click(InfoBlock.element('Relationships', rel))
-
-        # Containers Provider and Image Registry are string values
-        # Other rows in the table show the number of the given items
-        if rel in ['Containers Provider', 'Image Registry']:
-            assert val == InfoBlock.text('Properties', detailfield)
-        else:
-            val = int(val)
-            # There might be more than 1 page of items
-            if paginator.page_controls_exist():
-                paginator.results_per_page(1000)
-            assert len([r for r in list_tbl_image.rows()]) == val
-
-
-# 9868 #9869
-
-
-@pytest.mark.parametrize('rel',
-                         ['Containers Provider',
-                          'Routes',
-                          'Services',
-                          'Replicators',
-                          'Pods',
-                          'Nodes'])
-def test_projects_rel(provider, rel):
-    sel.force_navigate('containers_projects')
+    navigate_to(cls, 'All')
     tb.select('List View')
-    list_tbl_project = CheckboxTable(
-        table_locator="//div[@id='list_grid']//table")
-    ui_projects = [r.name.text for r in list_tbl_project.rows()]
-    mgmt_objs = provider.mgmt.list_project()
-
-    assert set(ui_projects).issubset(
-        [obj.name for obj in mgmt_objs]), 'Missing objects'
-
-    for name in ui_projects:
-        obj = Project(name, provider)
-
-        val = obj.get_detail('Relationships', rel)
-        if val == '0':
-            continue
-        obj.click_element('Relationships', rel)
-
-        try:
-            val = int(val)
-            assert len([r for r in list_tbl_project.rows()]) == val
-        except ValueError:
-            assert val == InfoBlock.text('Properties', 'Name')
+    list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
```

Comment:
Right, I added the locator to this test, in order to prevent caching issues that I ran into earlier.